.github/README.md

```diff
@@ -1,39 +1,47 @@
 [![wercker status](https://app.wercker.com/status/3e08a89169eeafef8ec020a9ceafe204/s/master "wercker status")](https://app.wercker.com/project/byKey/3e08a89169eeafef8ec020a9ceafe204) [![codecov](https://codecov.io/gh/DoSomething/gambit-campaigns/branch/master/graph/badge.svg)](https://codecov.io/gh/DoSomething/gambit-campaigns)
 
-# Gambit Campaigns
+# Gambit Content
 
-Gambit Campaigns is a [Gambit Conversations](https://github.com/dosomething/gambit-conversations) microservice for:
+Gambit Content fetches data from [Phoenix](https://github.com/DoSomething/phoenix) and Contentful to provide a Content API for use by [Gambit Conversations](https://github.com/dosomething/gambit-conversations).
 
-* Storing User Campaign activity from an inbound message, and posting it to [Rogue](https://github.com/dosomething/gambit-campaigns/wiki/rogue) via [Gateway-js](https://github.com/DoSomething/gateway-js).
+## API 
 
-* Querying Contentful for a list of active Keywords and Campaigns available running on Gambit.
+All endpoints require either a valid `x-gambit-api-key` header or `apiKey` query parameter.
 
-* Querying [Phoenix](https://github.com/DoSomething/phoenix) for Campaign data used in Gambit message templates.
+Endpoint                                       | Functionality                                           
+---------------------------------------------- | --------------------------------------------------------
+`GET /v1/broadcasts` | [Retrieve broadcasts](endpoints/broadcasts.md#retrieve-broadcasts)
+`GET /v1/broadcasts/:id` | [Retrieve a broadcast](endpoints/broadcasts.md#retrieve-broadcast)
+`GET /v1/campaigns/:id` | [Retrieve a campaign](endpoints/campaigns.md#retrieve-campaign)
+`GET /v1/contentfulEntries/` | [Retrieve Contentful entries](endpoints/contentfulEntries.md#retrieve-contentful-entries)
+`GET /v1/contentfulEntries/:id` | [Retrieve a Contentful entry](endpoints/contentfulEntries.md#retrieve-contentful-entry)
+`GET /v1/defaultTopicTriggers` | [Retrieve all additional default topic triggers](endpoints/defaultTopicTriggers.md)
+`GET /v1/topics` | [Retrieve topics](endpoints/topics.md#retrieve-topics)
```

Comment:
We should remove this route right? I get a `Cannot GET /v1/topics` when requesting it.

Reply:
yes! Thanks for catching this 🦅